manifests/plugin/snmp_agent.pp

```diff
@@ -0,0 +1,46 @@
+# https://wiki.opnfv.org/display/fastpath/SNMP+Agent+HLD
+class collectd::plugin::snmp_agent(
```

Comment:
can you please add puppet-strings documentation for the params?

Reply:
Added